src/cmds/pbs_attach.c

```diff
@@ -397,6 +372,34 @@ main(int argc, char *argv[])
 		 */
 		(void)putenv("MPICH_PROCESS_GROUP=no");
 
+		/*
+		 ** Load in the job environment variables needed for cgroups
+		 ** Check to see the file $PBS_NODEFILE.env exists
```

Comment:
Seems you are looking for a file called jobid.env, yet your comment mentions $PBS_NODEFILE.env.